NAMESPACE

```diff
@@ -1,57 +1,4 @@
 # Generated by roxygen2: do not edit by hand
 
-export(account_access_control_proxy)
-export(alerts)
-export(catalogs)
-export(cluster_policies)
-export(clusters)
-export(command_execution)
-export(configure)
-export(connections)
-export(current_user)
-export(dashboards)
-export(data_sources)
-export(dbfs)
-export(dbsql_permissions)
-export(experiments)
-export(external_locations)
-export(functions)
-export(git_credentials)
-export(global_init_scripts)
-export(grants)
-export(groups)
-export(instance_pools)
-export(instance_profiles)
-export(ip_access_lists)
-export(jobs)
-export(libraries)
-export(metastores)
-export(model_registry)
-export(permissions)
-export(pipelines)
-export(policy_families)
-export(providers)
-export(queries)
-export(query_history)
-export(recipient_activation)
-export(recipients)
-export(repos)
-export(schemas)
-export(secrets)
-export(service_principals)
-export(serving_endpoints)
-export(shares)
-export(statement_execution)
-export(storage_credentials)
-export(system_schemas)
-export(table_constraints)
-export(tables)
-export(token_management)
-export(tokens)
-export(users)
-export(volumes)
-export(warehouses)
-export(workspace)
-export(workspace_bindings)
-export(workspace_conf)
+export(DatabricksClient)
```

Comment:
Something doesn't look right here — I think you should be exporting all the new functions that you just created